modules/tinymce/src/core/test/ts/browser/dom/SerializerTest.ts

```diff
@@ -647,7 +647,7 @@ describe('browser.tinymce.core.dom.SerializerTest', () => {
   });
 
   it('BR at end of blocks', () => {
-    const ser = DomSerializer({ fix_list_elements: true });
+    const ser = DomSerializer({ fix_list_elements: true, remove_trailing_brs: true });
```

Comment:
Once we keep DomSerializer's defaulting behaviour for `remove_trailing_brs`, these tests no longer need to be changed.

Reply:
removed